packages/by-name/cli-release/package.nix

```diff
@@ -14,7 +14,7 @@
       install -D ${lib.getExe genpolicy} cli/genpolicy/assets/genpolicy
       install -D ${contrast.settings}/genpolicy-settings.json cli/genpolicy/assets/genpolicy-settings.json
       install -D ${contrast.rules}/genpolicy-rules.rego cli/genpolicy/assets/genpolicy-rules.rego
-      # TODO(burgerdev): cli/genpolicy/assets/allow-all.rego is insecure and deliberately omitted
+      install -D ${genpolicy.src}/src/kata-opa/allow-all.rego cli/genpolicy/assets/allow-all.rego
```

Comment:
My intent was that we never package an insecure policy in a released binary. Why do we need to package it now?

Reply:
This is only needed to run the tests. This commit will be dropped shortly before merging.

Reply:
> This commit will be dropped shortly before merging.

Done